public/static/app/perf/perf.js

```diff
@@ -659,10 +659,15 @@ mciModule.controller('PerfController', function PerfController(
           function(resp) {
             pointsPromise.then(function(outliers){
               const rejects = outliers.rejects;
-              let filtered = _.reject(resp.data, doc => _.contains(rejects, doc.task_id));
               $scope.allTrendSamples = new TrendSamples(resp.data);
-              $scope.filteredTrendSamples = new TrendSamples(filtered);
-
+              // Default filtered to all.
+              $scope.filteredTrendSamples = $scope.allTrendSamples;
+              if(rejects.length) {
+                let filtered = _.reject(resp.data, doc => _.contains(rejects, doc.task_id));
```

Comment:
[nit] Could this use 'const'? It doesn't look like filtered is being reassigned anywhere?

Reply:
for sure.